check-log/lib/check-log.go

```diff
@@ -171,7 +187,7 @@ func run(args []string) *checkers.Checker {
 			missingFiles = append(missingFiles, f)
```

Comment:
We should check whether `ctx` is alive in this `range` loop so that we can avoid extra `os.Stat` for files after `ctx` is over.